src/x509/cert.ts

```diff
@@ -146,6 +161,11 @@ export class x509Certificate {
     return ca;
   }
 
+  public extension(oid: string): x509Extension | undefined {
```

Comment:
nit pick rename? somehow this name makes me think it's a setter
```suggestion
  public getExtension(oid: string): x509Extension | undefined {
```

Reply:
I see we have `extensions()` already, also ok as is 👍